### PR TITLE
App and Client Integration

### PR DESCRIPTION
**Pull request by fussel178**
### Summary 

Integration changes to successfully integrate application and client.

### Details 

_coming soon_ :wink: 

### Additional information 

n/a

### CLA

- [x] I have signed the individual contributor's license agreement **and sent** it to the board of the WüSpace e. V. organization.

---
This change is [Reviewable](https://reviewable.io/reviews/wuespace/telestion-project-daedalus2/115)

### CODE REVIEW

File: application/conf/config.json [hunk 1]
```diff
@@ -129,17 +129,42 @@
...
+		  ]
+		}
+	  },
+	  {
+		"name": "Telecommand Sender",
+		"verticle": "de.wuespace.telestion.project.daedalus2.mavlink.TelecommandSender",
+		"magnitude": 1,
+		"config": {
+		  "inAddress": "tc-sender",
+		  "outAddress": "bla",
```
Comment (cb0s) [inline application/conf/config.json:155]: Maybe this should be changed to something serious?

**Review by cb0s — CHANGES_REQUESTED** [review]
In general this looks good, but there is one thing I would like to change :)

**Review by fussel178 — COMMENTED** [review]
# :rofl:

**Review by pklaschka — CHANGES_REQUESTED** [review]
> Reviewed 5 of 9 files at r1, 12 of 13 files at r3, 1 of 1 files at r4, 3 of 3 files at r5.
> *[Reviewable](https://reviewable.io/reviews/wuespace/telestion-project-daedalus2/115)* status: all files reviewed, 2 unresolved discussions (waiting on @fussel178)

___
*[client/package.json, line 77 at r5](https://reviewable.io/reviews/wuespace/telestion-project-daedalus2/115#-Md-cHaCEsRTU0PzhN6F:-Md-cHaCEsRTU0PzhN6G:bz85gw0) ([raw file](https://github.com/wuespace/telestion-project-daedalus2/blob/3e9403293548c194f006c9ef15d9b0d53e126129/client/package.json#L77)):*
> ```JSON
> 		"@wuespace/telestion-client-types": "^0.16.0",
> 		"@wuespace/vertx-mock-server": "^0.16.0",
> 		"node-sass": "^6.0.1",
> ```

Any reason why we use `node-sass` (which is deprecated) instead of `sass`?

**Review by fussel178 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/wuespace/telestion-project-daedalus2/115)* status: all files reviewed, 2 unresolved discussions (waiting on @fussel178 and @pklaschka)

___
*[client/package.json, line 77 at r5](https://reviewable.io/reviews/wuespace/telestion-project-daedalus2/115#-Md-cHaCEsRTU0PzhN6F:-Md1jPbV2VT97LGZGVQV:b-u12r9l) ([raw file](https://github.com/wuespace/telestion-project-daedalus2/blob/3e9403293548c194f006c9ef15d9b0d53e126129/client/package.json#L77)):*
<details><summary><i>Previously, pklaschka (Pablo Klaschka) wrote…</i></summary><blockquote>

Any reason why we use `node-sass` (which is deprecated) instead of `sass`?
</blockquote></details>

Because the guys from react-scripts recommend `node-sass` on their page (https://create-react-app.dev/docs/adding-a-sass-stylesheet) and I have no idea, which version or which package is the right one.
Have we not already support for SCSS from the `tc-cli`?

**Review by fussel178 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/wuespace/telestion-project-daedalus2/115)* status: all files reviewed, 2 unresolved discussions (waiting on @fussel178 and @pklaschka)

___
*[client/package.json, line 77 at r5](https://reviewable.io/reviews/wuespace/telestion-project-daedalus2/115#-Md-cHaCEsRTU0PzhN6F:-Md1oeWw3OS6gfOX29WD:b7l91ba) ([raw file](https://github.com/wuespace/telestion-project-daedalus2/blob/3e9403293548c194f006c9ef15d9b0d53e126129/client/package.json#L77)):*
<details><summary><i>Previously, fussel178 (Ludwig Richter) wrote…</i></summary><blockquote>

Because the guys from react-scripts recommend `node-sass` on their page (https://create-react-app.dev/docs/adding-a-sass-stylesheet) and I have no idea, which version or which package is the right one.
Have we not already support for SCSS from the `tc-cli`?
</blockquote></details>

I've removed `node-sass` and installed `sass`.

**Review by fussel178 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/wuespace/telestion-project-daedalus2/115)* status: all files reviewed, 2 unresolved discussions (waiting on @fussel178 and @pklaschka)

___
*[client/package.json, line 77 at r5](https://reviewable.io/reviews/wuespace/telestion-project-daedalus2/115#-Md-cHaCEsRTU0PzhN6F:-Md1pjjcETBN2dKiWY4v:b-896fix) ([raw file](https://github.com/wuespace/telestion-project-daedalus2/blob/3e9403293548c194f006c9ef15d9b0d53e126129/client/package.json#L77)):*
<details><summary><i>Previously, fussel178 (Ludwig Richter) wrote…</i></summary><blockquote>

I've removed `node-sass` and installed `sass`.
</blockquote></details>

Done.

**Review by pklaschka — APPROVED** [review]
:lgtm:

> Reviewed 2 of 2 files at r6.
> *[Reviewable](https://reviewable.io/reviews/wuespace/telestion-project-daedalus2/115#-:-Md22Cwb2Tg30d5INdbk:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @fussel178)

___
*[client/package.json, line 77 at r5](https://reviewable.io/reviews/wuespace/telestion-project-daedalus2/115#-Md-cHaCEsRTU0PzhN6F:-Md221_1Dl-nus4xw4Go:b6upsof) ([raw file](https://github.com/wuespace/telestion-project-daedalus2/blob/3e9403293548c194f006c9ef15d9b0d53e126129/client/package.json#L77)):*
<details><summary><i>Previously, fussel178 (Ludwig Richter) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

I at least believe they should be compatible, so it should probably just work :wink: Thank you!

**Pull request merged**